### DISCUSSION TITLE
about javafx

### DISCUSSION THREAD

**Discussion by EshanDeSilva** [EshanDeSilva/Calculator#10] [Q&A] [answered]
How to solve this ,Error: JavaFX runtime components are missing, and are required to run this application(When run this program)

**Comment by VishalSandakelum** [comment, marked as answer]
If you are using build up to Java 8 you should add Java SDK lib PATH and then this error will be resolved. If you build this project using maven. You can edit configurations and add VM Options to as below

--module-path
"ADD YOUR SDK LIB PATH HERE"
--add-modules
javafx.controls,javafx.fxml
--add-exports
javafx.graphics/com.sun.javafx.sg.prism=ALL-UNNAMED
